### DISCUSSION TITLE
Need Help

### DISCUSSION THREAD

**Discussion by aliaktas-tr** [TheAlphamerc/flutter_twitter_clone#152] [Q&A]
Hi @TheAlphamerc  

I need your help with two subjects if it's possible?

1)  How can I update the user in the feed model at old tweets?
2) How can I add one specific user to the new users following list automatically?

Thanks

**Comment by TheAlphamerc** [comment]
Hi @a-l-iak , You can find your answer for first question [here](https://github.com/TheAlphamerc/flutter_twitter_clone/issues/80)

**Reply by TheAlphamerc** [reply]
Answer to your second question in below.

If you want to know how to follow/unfollow other user you can have a look on below code.

https://github.com/TheAlphamerc/flutter_twitter_clone/blob/5dc2e226332bc904765d0022b58ee3823d849c9e/lib/state/authState.dart#L423
